src/settings.rs

```diff
@@ -170,15 +170,15 @@ impl Settings {
 
         let tile_sizes_str = crate::engine::AVAILABLE_TILE_SIZES
             .iter()
-            .map(|num| num.to_string())
+            .map(std::string::ToString::to_string)
```

Comment:
This is almost twice as long. I'm okay with this if we do `use std::string::ToString` at the top of the function/module and only use `ToString::to_string` in the `map` call).

Reply:
Will do it.